src/devices/oregon_scientific.c

```diff
@@ -142,6 +142,26 @@ static unsigned long long cm180i_total(uint8_t const *msg)
     return val;
 }
 
+static unsigned int swap_nibbles(unsigned char byte) {
+ return (((byte&0xf)<<4) | (byte>>4));
+}
+
+static unsigned int cm160_power(uint8_t const *msg)
```

Comment:
remove both cm160_ functions and put the code directly in the block below.

Reply:
Resolved